types/provision/index.ts

```diff
@@ -21,13 +21,18 @@ export enum ClusterCreationStep {
   DETAILS,
 }
 
-export type NewClusterConfig = {
+export enum ImageRepository {
+  GITHUB = 'github',
+  AWS = 'aws',
+}
```

Comment:
We only need these values:
```suggestion
export enum ImageRepository {
  GIT = 'git',
  ECR = 'ecr',
}
```